charts/exerciseset2/templates/exerciseSet.yaml

```diff
@@ -0,0 +1,95 @@
+---
+apiVersion: kubeteach.geberl.io/v1alpha1
+kind: ExerciseSet
+metadata:
+  name: set2
+spec:
+  taskDefinitions:
+    - name: task01
+      taskDefinitionSpec:
+        taskSpec:
+          title: "Create namespace"
+          description: "Create a new namespace with the name kubeteach"
+          helpURL: "https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/"
+        taskCondition:
+          - apiVersion: v1
+            kind: Namespace
+            name: "kubeteach"
+        points: 5
+    - name: task02
+      taskDefinitionSpec:
+        taskSpec:
+          title: "Create a Persistent Volume Claim"
+          description: "Create a Persistent Volume Claim (PVC) in the cluster with the name myfirstpvc in the task01's namespace, request 1Gi storage with RWO access mode"
```

Comment:
I would also check the RWO mode in the `resourceCondition` as it is mentioned in the description.